src/language-js/parse/babel.js

```diff
@@ -77,7 +77,6 @@ function parseWithOptions(parseMethod, text, options) {
   /** @type {import("@babel/parser").parse | import("@babel/parser").parseExpression} */
   const parse = require("@babel/parser")[parseMethod];
   const ast = parse(text, options);
-  // @ts-expect-error
```

Comment:
https://github.com/babel/babel/pull/13653

Reply:
I recall there is something else missing, but I can't remember now.

Reply:
Oh, right, `attachComment` seems missing in  `ParserOptions` for 
 parseExpression, https://github.com/prettier/prettier/runs/3258106700#step:6:6, maybe you can fix that too.

Reply:
Oh I didn't know that! Thanks!